proto/binlogdata.proto

```diff
@@ -353,6 +353,10 @@ message FieldEvent {
   repeated query.Field fields = 2;
   string keyspace = 3;
   string shard = 4;
+
+  // Field numbers in the gap between shard (4) and enum_set_string_values
```

Comment:
These comments in this file are unrelated to this PR but I wanted to get them into main after the field number gap came up in a discussion.